apps/webpack.js

```diff
@@ -39,15 +39,19 @@ var baseConfig = {
       http: 'stream-http',
     }
   },
-  sassLoader: {
-    includePaths: [path.resolve(__dirname, '..', 'shared', 'css')]
-  },
   module: {
-    loaders: [
+    rules: [
       {test: /\.json$/, loader: 'json-loader'},
       {test: /\.ejs$/, loader: 'ejs-compiled-loader'},
       {test: /\.css$/, loader: 'style-loader!css-loader'},
       {test: /\.scss$/, loader: 'style-loader!css-loader!sass-loader'},
+      {
+        test: /\.css$/,
```

Comment:
Should this replace the `/\.css$/` rule above, or should the above rule explicitly exclude the shared directory?  Also, it's a bummer to treat our css files differently depending on which directory they're in.  Since scss is strictly a superset of css, is it safe to add sass-loader for all of our css files instead of just for this directory?

Reply:
I did a double-take at this as well 😄.  This ends up being a direct translation of the `sassLoader` which was removed [here](https://github.com/code-dot-org/code-dot-org/pull/19706/files#diff-c12a5734510ba76bad129dc78743af96L42).

https://github.com/code-dot-org/code-dot-org/blob/1423cd6e5e2ccc9ff1b64602a0f2014bb657a453/apps/webpack.js#L42-L44

We can combine the two rules, but my plan for this PR to match the existing functionality as close as possible.  Followup PR?

Reply:
Followup PR sounds good :+1:

Reply:
Actually there's no reason this can't happen before the webpack upgrade!  PR here: https://github.com/code-dot-org/code-dot-org/pull/19806.